packages/service-integration/Dockerfile

```diff
@@ -0,0 +1,81 @@
+ARG PYTHON_VERSION="3.9.12"
+FROM python:${PYTHON_VERSION}-slim-buster as base
+
+LABEL maintainer=pcrespov
+
+RUN set -eux && \
+  apt-get update && \
+  apt-get install -y --no-install-recommends \
+  gosu && \
+  rm -rf /var/lib/apt/lists/* && \
+  # verify that the binary works
+  gosu nobody true
+
+
+# simcore-user uid=8004(scu) gid=8004(scu) groups=8004(scu)
+ENV SC_USER_ID=8004 \
+  SC_USER_NAME=scu \
+  SC_BUILD_TARGET=base \
+  SC_BOOT_MODE=default
+
+RUN adduser \
+  --uid ${SC_USER_ID} \
+  --disabled-password \
+  --gecos "" \
+  --shell /bin/sh \
+  --home /home/${SC_USER_NAME} \
+  ${SC_USER_NAME}
```

Comment:
I guess all this is not required?

Reply:
ufff ... the dangers of copy&paste&night-coding :-) thx!